exercises/exercise_4/README.md

```diff
@@ -48,13 +48,14 @@ By default, the vote and results services let you vote between Dogs and Cats. Ho
     ```
     </details>
 
-2. Before setting the parameter, run `docker app inspect` to see what happens.
+2. Before setting the parameter, run `docker app build voting-app -t <your-hub-username>/voting-app.dockerapp && docker app inspect --pretty` to see what happens.
```

Comment:
The inspect command should be `docker app inspect <your-hub-username>/voting-app.dockerapp`

Reply:
fixed